canonical-web-and-design/certification.ubuntu.com
# Simplify codebase

By removing useless (and confusingly named) oneline API functions, and removing the pointless blueprint, and `url_for` functions.

QA
--

`./run clean && ./run`

And browse the site, check it works as well as it did before.
